hc/front/tests/test_add_telegram.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+from django.test.utils import override_settings
+from hc.api.models import Channel
+from hc.api.transports import SMS
+from hc.test import BaseTestCase
+from mock import patch
+
+
+class AddSMSTestCase(BaseTestCase):
+
+    def test_it_shows_instructions(self):
+        self.client.login(username="alice@example.org", password="password")
+        r = self.client.get("/integrations/add_telegram/")
+        self.assertContains(r, "@hckaliiBot", status_code=200)
+
+    def test_it_adds_sms(self):
```

Comment:
@philophilo is this the right test name for this test? Wouldn't `test_it_adds_telegram` be better?

Reply:
correcting it